gossip/src/crds.rs

```diff
@@ -565,6 +593,7 @@ impl Crds {
         match value.value.data() {
             CrdsData::ContactInfo(_) => {
                 self.nodes.swap_remove(&index);
+                self.notify_remove_node(&value.value.pubkey());
```

Comment:
All CrdsData is being notified during update, while only contact info is notified during remove.
Could be intended behavior.

Reply:
so CrdsData is only being updated on ContactInfo as well. See: https://github.com/gregcusack/solana/blob/bd9dbe00404343a01072a75970a5520dbb426c21/gossip/src/crds.rs#L256-L258

this^ function does the check for `ContactInfo` instead of the check being done in `crds.insert()`.